pytorch_lightning/accelerators/accelerator.py

```diff
@@ -118,11 +118,15 @@ def clip_gradients(self, optimizer, clip_val=None):
         self._clip_gradients(optimizer, grad_clip_val)
 
     def _clip_gradients(self, optimizer: Optimizer, grad_clip_val: Union[float, int], norm_type: float = 2.0):
+        clip_algorithm = self.trainer.gradient_clip_algorithm
         if self.trainer.amp_backend:
-            self.trainer.precision_connector.backend.clip_gradients(grad_clip_val, optimizer, norm_type)
+            self.trainer.precision_connector.backend.clip_gradients(grad_clip_val, clip_algorithm, optimizer, norm_type)
         else:
             model = self.trainer.get_model()
-            torch.nn.utils.clip_grad_norm_(model.parameters(), max_norm=grad_clip_val, norm_type=norm_type)
+            if clip_algorithm == 'value':
```

Comment:
Let's use an LightningEnum there .

Reply:
Thank you for review. I'll change my code.